resourcemgr/resourcemgr.c

```diff
@@ -2939,6 +2939,7 @@ TSS2_RC InitResourceMgr( int debugLevel)
     rval = GetCommands( resMgrSysContext, &supportedCommands );
     if( rval != TPM_RC_SUCCESS )
     {
+        rval += TSS2_RESMGR_ERROR_LEVEL;
```

Comment:
Agree, but I'm realizing that this will also add the wrong layer indicator for errors returned from further down the stack. Honestly I'm tempted to just move the 'getcommands' function into the resourcemgr directly since it's not used anywhere else in the code. That will make this much more simple. Agree / disagree?